R/net.mod.init.R

```diff
@@ -48,15 +48,15 @@ initialize.net <- function(x, param, init, control, s) {
     # Network Parameters ------------------------------------------------------
     dat$nwparam <- list(x[-which(names(x) == "fit")])
     groups <- length(unique(get.vertex.attribute(nw, "group")))
-    dat$param$groups <- groups
+    dat <- set_param(dat, "groups", groups)
 
     # Nodal Attributes --------------------------------------------------------
 
     # Standard attributes
     num <- network.size(nw)
     dat$attr$active <- rep(1, num)
```

Comment:
why not use set_attr for active?

Reply:
@smjenness set_attr is based on length of active. Initial setting of active has to be using the initial method, unless we remove the attr length check.

Reply:
ah, right. let's add an `override.length.check` arg to set_attr that defaults to FALSE, but would be TRUE in this case.

Reply:
on it

Reply:
Updated by Adrien.